code/game/objects/items/handcuffs.dm

```diff
@@ -539,3 +539,145 @@
 	. = ..()
 	if(effectReference)
 		QDEL_NULL(effectReference)
+
+/**
+ * Deathcuffs
+ * Can be worn in glove slot to give an alert to medbay & cargo when you die off-station.
+ * Acts as a GPS when worn as a bracelet.
+ * Can otherwise be used as normal handcuffs
```

Comment:
why

Reply:
Why what, the comments? All other handcuff variants have them too.

Reply:
why can they also be used as handcuffs, what are you trying to achieve? what game role are you fufilling what niche is this object supposed to create?

Reply:
a new miner item that helps miners be recovered when they die that doubles as a stealth antag item that allows you to capture people easier. also allows you to capture ashwalkers and take them in.

Reply:
what are you the lorax that speaks for wallemations? nobody asked.

Reply:
A lot of that is explained in the "why it's good for the game" section. 
Lavaland is probably one of the worst places to die, and I believe that adding a tool that can be bought for mining points to give you a _slightly higher chance_ at being revived is a fair way to reward players without entirely removing the risk.
If you feel it's too cheap for what it does, I can always raise the point cost.

Additionally the reason it requires handcuffs to make is to get the miner to interact with the bureaucracy of the station before shoving off back to lavaland instantly upon buying them. Rewarding player interaction n stuff
As for why they're handcuffs:
Honestly just because it's cool.
Making cable ties is easy enough that I felt this wasn't too powerful of a tool. Besides, we could always nerf the breakout time.

Reply:
> a new miner item that helps miners be recovered when they die that doubles as a stealth antag item that allows you to capture people easier. also allows you to capture ashwalkers and take them in.

cable cuffs are honestly better, can be made round start, and are stealthy since you can just have cable on you and make them whenever you need. i wouldnt really say spending 2000 points for a pair of handcuffs is really good at all

Reply:
> Additionally the reason it requires handcuffs to make is to get the miner to interact with the bureaucracy of the station

Handcuffs being a requirement won't force player interaction like you think it will, as miners will just print handcuffs from the cargo autolathe, which they have roundstart access to.

Handcuffs being "cool" isn't a good enough reason to justify the confusion this adds imo.

Reply:
If I removed them from the handcuff typetree and just made them plain items that go on your glove slot would that be better?
That way they won't function as handcuffs, just the GPS & Lifeline functions

Reply:
You could have handcuffs be part of the recipe but leave the resulting item without cuff functionality. This would fix the ambiguity of the item while still requiring the cuffs (and still give antag miners an alibi about having cuffs, if that's an intention).

Why are cuffs in the autolathe?

Reply:
Seeing as they're no longer handcuffs, I'm gonna resolve this.